components/gcp/container/component_sdk/python/kfp_component/google/ml_engine/_train.py

```diff
@@ -15,45 +15,60 @@
 from fire import decorators
 from ._create_job import create_job
 
+
 @decorators.SetParseFns(python_version=str, runtime_version=str)
-def train(project_id, job_id_output_path, python_module=None, package_uris=None, 
-    region=None, args=None, job_dir=None, python_version=None, 
-    runtime_version=None, master_image_uri=None, worker_image_uri=None, 
-    training_input=None, job_id_prefix=None, job_id=None, wait_interval=30):
+def train(project_id,
+          job_id_output_path,
+          job_dir_output_path,
+          python_module=None,
```

Comment:
Not urgent but weird... I vaguely remember the google style should be `python_module = None`. What's the linter are you using for this file?

Reply:
I used go/pyformat, and it didn't add the space. Guessing either the space is not a hard requirement or it's only enforced when there's type hints?

Reply:
SG.